repo2docker/buildpacks/conda/__init__.py

```diff
@@ -63,7 +63,7 @@ def get_build_scripts(self):
 
         All scripts here should be independent of contents of the repository.
 
-        This sets up through `install-miniforge.bash` (found in this directory):
+        This sets up through `install-micromamba.bash` (found in this directory):
```

Comment:
We've done this before, and should have fixed it then - it doesn't really make sense to keep renaming this file as we change some implementation details inside it. Maybe change it to `install-conda.bash` or `install-base-env.bash`? Whether we use conda, miniconda, micromamba, etc. should not mean renaming this file anymore.

Reply:
Done!